### DISCUSSION TITLE
Module not found: Can't resolve 'next/Image'

### DISCUSSION THREAD

**Discussion by lokeshpatchala** [vercel/next.js#36764] [Help] [answered]
When I am trying to build my application vercel I am getting the error  Module not found: Can't resolve 'next/Image', I am using the latest version of next.js and my version number is 12.1.5

**Comment by samuelebenitez** [comment]
Hi @lokeshpatchala! I'm having the same issue. I can't find where is the error ¿Have u tried deleting node_modules? 
I've been searching and update to the last version of Next doesn't work for me. ¿Could u find a solution? Thanks

**Reply by fbotu** [reply]
I had a similar of: Module not found: Can't resolve 'next/legacy/image' in '/vercel/path0/pages'
Deleting node_modules and yarn install seemed to fix it.

Thanks @samuelebenitez

**Comment by lokeshpatchala** [comment, marked as answer]
Hi Samuel,
Yeah my issue got resolved in my case the made the wrong import I used
next/image instead of next/Image
